create_infotexts.py

```diff
@@ -195,6 +195,8 @@ def main():
         img = CypernImage()
         img.generate_list_of_stripped_keywords(metadata[fotonr]["Nyckelord"])
         img.create_commons_filename(metadata[fotonr])
+        img.add_cat_archaeological_exhibition_based_on_description(metadata[fotonr]["Beskrivning"])
```

Comment:
Would it make more sense to implement both these in the same function? `img.add_special_cats_based_on_descritpion(metadata[fotonr]["Beskrivning"])`